e2e/wdio/core/tests/wizard.e2e-spec.ts

```diff
@@ -0,0 +1,232 @@
+
+        expect(getElementClass(dialogWizard + buttonsBar + ':nth-child(2) ' + button)).toContain('is-disabled');
+        expect(isElementClickable(dialogWizard + buttonsBar + ':nth-child(2) ' + button)).toBe(false, 'button is clickable');
+        setValue(fullNameInput, fullName);
+        setValue(firstAdressInput, firstAdress);
+        expect(getElementClass(dialogWizard + buttonsBar + ':nth-child(2) ' + button)).not.toContain('is-disabled');
+        expect(isElementClickable(dialogWizard + buttonsBar + ':nth-child(2) ' + button)).toBe(true, 'button is not clickable');
+
+        for (let i = 0; i < firstAdressLength; i++) {
+            sendKeys('Backspace')
+        }
+        expect(getElementClass(dialogWizard + buttonsBar + ':nth-child(2) ' + button)).toContain('is-disabled');
+        expect(isElementClickable(dialogWizard + buttonsBar + ':nth-child(2) ' + button)).toBe(false, 'button is clickable');
+    });
+
+    it('should check confirmation changing payment type in branching example', () => {
+        click(branchingExample + button);
+        click(wizard + nextStep);
+        click(radioButton);
+        expect(getAttributeByName(radioButton, 'aria-checked')).toBe('true', 'radio button is not selected');
+        click(radioButton, 1);
+        expect(isElementDisplayed(dialogContainer)).toBe(true, 'dialog container did not open');
+        click(cancelButton);
+        expect(getAttributeByName(radioButton, 'aria-checked')).toBe('true', 'focus dissapeared');
+        click(radioButton, 1);
+        click(continueButton);
+        expect(getAttributeByName(radioButton, 'aria-checked', 1)).toBe('true', 'focus did not change');
+    });
+
+    it('should check navigation in mobile example', () => {
+        click(mobileExample + nextStep);
+        expect(getElementClass(mobileExample + step, 1)).toContain('step--current', 'you are not moved to the second step');
+        click(mobileExample + stepContainer);
+        expect(getElementClass(mobileExample + step, 0)).toContain('step--current', 'you are not moved to the first step');
+        click(mobileExample + nextStep);
+        expect(getElementClass(mobileExample + step, 1)).toContain('step--current', 'button did not direct you to the step 2');
+    });
+
+    it('should check compliting form for mobile example', () => {
+        const stepsLength = getElementArrayLength(mobileExample + step) / 2;
+        for (let i = 0; i < stepsLength; i++) {
+            expect(getElementClass(mobileExample + step, i)).toContain('step--current');
+            click(mobileExample + nextStep);
+        }
+    });
+
+    it('should check orientations', () => {
+        wizardPage.checkRtlSwitch();
+    });
+
+    it('should check examples visual regression', () => {
+        wizardPage.saveExampleBaselineScreenshot();
+        expect(wizardPage.compareWithBaseline()).toBeLessThan(5);
+    });
+
+    function checkReOpen(section: string, block: string) {
```

Comment:
functions should have a typeof def (i.e. void, number, string, etc)

function checkReOpen(section: string, block: string): void {
...
}